content/zh-tw/certificates.md

```diff
@@ -1,77 +1,76 @@
+  * [Let's Encrypt Authority X1（與 IdenTrust 交互簽名）](/certs/lets-encrypt-x1-cross-signed.pem.txt)
+    * [Let's Encrypt Authority X1（由 ISRG Root X1 簽發）](/certs/letsencryptauthorityx1.pem.txt)
+
+# 交互簽名
 
-# 交叉簽名
+我們的中間憑證 "Let's Encrypt Authority X3" 具有一對公鑰和私鑰，
+我們使用金鑰中的私鑰替所有終端憑證（也稱為子葉憑證）簽名，這也就是我們頒發給你伺服器的憑證。
 
-我們的中級憑證“Let's Encrypt Authority X3”代表了一對公鑰/私鑰組合。這對密鑰中的私鑰為所有終端實體簽名
-生成憑證（也稱為葉憑證），即我們頒發給您用於伺服器使用的憑證。
+我們的中間憑證是由 ISRG Root X1 根憑證所簽發。因為我們還是個非常新的憑證頒發機構，ISRG Root X1 尚未受到大多數的瀏覽器的信任。為了使我們頒發的憑證被廣為信任，我們向一個已受主流瀏覽器信任的根憑證 IdenTrust，交互簽名後產生了我們的中間憑證。精確地說，IdenTrust 使用他們的 "DST Root CA X3"（現在被稱為 "TrustID X3 Root"）與我們交互簽名後產生了我們的中間憑證。[你可以從 identrust.com 網站下載 "TrustID X3 Root"](https://www.identrust.com/support/downloads)（或是你也可以在這裡下載副本：[.pem](/certs/trustid-x3-root.pem.txt) 及 [.p7b](/certs/trustid-x3-root.p7b)）。
 
-我們的中級憑證由ISRG Root X1簽署。 由於我們還是比較新的憑證頒發機構，ISRG Root X1在大多數瀏覽器中尚未受到信任。為了使我們頒發的憑證被大多數瀏覽器信任， 我們請求IdenTrust，一個根憑證已經受到全部主流瀏覽器信任的憑證頒發機構，和我們交叉簽名了我們的中級憑證。具體來說， IdenTrust使用他們的"DST Root CA X3"（現在稱為“TrustID X3 Root”）交叉簽名了我們的中級憑證。 [從identrust.com下載"TrustID X3 Root"憑證](https://www.identrust.com/support/downloads) (或者，您也可以在此處下載副本： [.pem](/certs/trustid-x3-root.pem.txt), [.p7b](/certs/trustid-x3-root.p7b)).
+這表示我們現在有兩種中間憑證。一種由 DST Root CA X3 根憑證所簽發，而另一種由 ISRG Root X1 根憑證所簽發。區分這兩種憑證最簡單的方法，就是查看憑證的簽發者 (Issuer) 欄位。
 
-這意味著現在有兩張憑證都可以代表我們的中級憑證。 一張由DST Root CA X3簽發，另外一張由ISRG Root X1簽發。區分這兩張憑證的最簡單方法是查看其簽發人（Issuer）欄位。
+當你在設定網頁伺服器時，伺服器管理者不只需要設定終端憑證，也需要設定中間憑證；以幫助瀏覽器透過信任鏈，驗證終端憑證是否被受信任的根憑證所簽發。幾乎所有的伺服器管理者都會選擇使用由 "DST Root CA X3" 根憑證，以及 "Let’s Encrypt Authority X3" 中間憑證所組成的憑證信任鏈。我們建議你使用 Let's Encrypt 所開發的軟體 [Certbot](https://certbot.org)，替你無縫地設定憑證信任鏈。
 
-在配置網路伺服器時，配置人員不僅需要配置終端憑證，也需要配置中級憑證以幫助瀏覽器通過信任鏈（Trust Chain）驗證終端憑證由被瀏覽器信任的根憑證籤發。幾乎所有的伺服器配置人員都會選擇使用由“DST Root CA X3”簽發的”Let’s Encrypt Authority X3“組成憑證信任鏈。我們推薦的憑證頒發軟體[Certbot](https://certbot.org)將無縫化配置憑證信任鏈。
+下圖表示憑證之間的關係：
 
-下圖用視覺方式詮釋了憑證之間的關係：
+<img src="/certs/isrg-keys.png" alt="ISRG 憑證關係圖">
 
-<img src="/certs/isrg-keys.png" alt="ISRG憑證關係圖">
+# OCSP 憑證
 
-# OCSP簽名憑證
+為避免我們將 ISRG 根憑證的私鑰放到線上伺服器中，因此我們使用 OCSP 憑證的私鑰替 Let's Encrypt 中間憑證簽署 OCSP 回應。OCSP 回應中會自動將這張憑證的複本包含在其中，因此使用者不需要需進行任何操作。這裡放上此憑證供你參考。
 
-為避免我們將ISRG根憑證上線以簽署Let's Encrypt Authority的中級憑證OCSP回復，我們建立了這張憑證用於簽發OCSP回復。OCSP回覆中自動包括了該憑證的副本，所以訂戶無需進行任何操作。此憑證僅供參考。
-* [ISRG Root OCSP X1 (由ISRG Root X1簽署)](/certs/isrg-root-ocsp-x1.pem.txt)
+* [ISRG Root OCSP X1 （由 ISRG Root X1 簽發）](/certs/isrg-root-ocsp-x1.pem.txt)
 
 # 憑證透明度
 
-我們努力對我們的運營及憑證頒發過程保持透明。 我們在頒發憑證時會將該憑證發送到[憑證透明度日誌](https://www.certificate-transparency.org/)。 您可以通過以下連結查看所有Let's Encrypt頒發的憑證：
+我們努力使 Let's Encrypt 的運作及憑證頒發過程保持透明。我們在頒發憑證時，會將憑證透過[憑證透明度](https://www.certificate-transparency.org/)系統紀錄下來。你可以通過以下連結查看所有 Let's Encrypt 所頒發過的憑證：
 
-* [由Let's Encrypt Authority X1發布](https://crt.sh/?Identity=%25&iCAID=7395)
-* [由Let's Encrypt Authority X3發布](https://crt.sh/?Identity=%25&iCAID=16418)
+* [由 Let's Encrypt Authority X1 頒發](https://crt.sh/?Identity=%25&iCAID=7395)
```

Comment:
There's an inconsistency between 69/70 (spacing)

Reply:
Thank for your careful check.